### DISCUSSION TITLE
使用vc-overlay-echarts 可以显示图表，但是无法用鼠标事件交互，怎么样进行交互？

### DISCUSSION THREAD

**Discussion by fuchunfen** [zouyaoji/vue-cesium#171] [Q&A]
使用vc-overlay-echarts 可以显示图表，但是无法用鼠标事件交互，怎么样进行交互？比如文档里给的那个例子也没有交互事例，如果要添加tooltip要怎么做？

**Comment by zouyaoji** [comment]
这是因为默认将它的 pointer-events 属性设为了 none了。你重置一下这个就可以了。

你试试能不能重置呢，不能重置就提个issue我在下一个版本开放一下。

**Reply by fuchunfen** [reply]
确实修改pointer-events可以和图表进行交互了，地图的鼠标事件会失效，不能移动和缩放，图表和地图是不是只能选一个交互？

**Reply by zouyaoji** [reply]
是的哈。这2个东西实际上就是 2个 div 强行叠加到一起了
